extensions/vscode-vue-language-features/README.md

```diff
@@ -158,6 +158,27 @@ defineProps<Props>()
 </script>
 ```
 
+### Custom File Extensions
+
+Syntax highlighting and intellisense can be applied to additional file extensions beyond just `.vue`. This will need to be configured in three different places for full support.
+
+In VS Code settings for the Volar extension add any additional extensions you need to `Additional Extensions`. Such as `.ext`.
+
+In your tsconfig.json file you will need to make sure your custom extension is included by TypeScript. If you have an include value for `./src/*.vue` then you would want to add an include for `./src/*.ext` as well. Next add a new key to the root of tsconfig.json called `vueCompilerOptions` with a child key of `extensions`. This should be an array that contains the `.vue` extension as well as your custom `.ext` extension.
+
+```json
+"include": [
+  "./src/*.ts",
+  "./src/*.vue",
+  "./src/*.ext",
+],
+"vueCompilerOptions": {
+  "extensions": [".vue", ".ext"]
```

Comment:
You don't need to change `vueCompilerOptions.extensions`, it should already add by https://github.com/johnsoncodehk/volar/pull/2035/files#diff-3356ce53da3e37d797343c3929442060ec0f344644d1ba892274169f338e4ae0R126-R131.

Reply:
You are correct. Updated the README file to reflect this.